flink/v1.17/flink/src/test/java/org/apache/iceberg/flink/TestFlinkCatalogTable.java

```diff
@@ -357,6 +358,213 @@ public void testAlterTableWithPrimaryKey() throws TableNotExistException {
     Assert.assertEquals(properties, table("tl").properties());
   }
 
+  @Test
+  public void testAlterTableProperties() throws TableNotExistException {
```

Comment:
This method is very similar to testAlterTable(), they're doing the same thing, right?

Reply:
Yes, maybe I should remove the original testAlterTable()?

Reply:
yes, I think so.